### ISSUE TITLE
vim motions "v a t" or "v i t" is not working

### ISSUE THREAD

**Issue by m1guer** [zed-industries/zed#16034]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

1. install zed.dev
2. enable vim mode.
3. jsx/html file tryna use vim motions.
4. not working and has no logging.

### Environment

  System:
    OS: Linux 6.10 Arch Linux
    CPU: (12) x64 AMD Ryzen 5 4500 6-Core Processor
    Memory: 10.86 GB / 15.42 GB
    Container: Yes
    Shell: 5.9 - /usr/bin/zsh

### If applicable, add mockups / screenshots to help explain present your vision of the feature

![image](https://github.com/user-attachments/assets/b7576c50-a630-4f3f-9de3-88a2720c4b75)

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by m1guer** [comment]
already have, but after the last update stop working.

**Comment by mkykode** [comment]
@m1guer Yes, it's all messed up now. It seems to only work on curly braces, but HTML goes all over the place.

**Comment by m1guer** [comment]
that's sad

**Comment by benmooo** [comment]
this is catastrophe!

**Comment by helgemahrt** [comment]
@m1guer does this issue still reproduce? I just tried using the motions on a simple HTML file and they seem to work fine.
